### PR TITLE
Paused markets bugs

### PR DESCRIPTION
**Pull request by brossetti1**
there are some inconsistencies with the timer and a closed markets. We are simplifying this functionality to just rely on the synthetix closed markets contract calls used from `useGetFuturesMarkets` and `useIsMarketClosed`

## Description
- [X] removing the timer, replacing with "closes/opens soon" and only showing approx 12 hours before market
- [X] removing the translation from the `MarketOverlay` that presents the timer.

## Related issue
https://github.com/Kwenta/kwenta/issues/712

## Motivation and Context
timer is not synced right and theres inconsistencies with timer data.

## How Has This Been Tested?
forced markets to closed and tested paused badges - default shows no timer.

## Screenshots (if appropriate):
Screen Shot 2022-04-19 at 2 40 09 PM
Screen Shot 2022-04-19 at 2 40 21 PM
Screen Shot 2022-04-19 at 2 40 26 PM
Screen Shot 2022-04-19 at 2 52 02 PM
Screen Shot 2022-04-19 at 2 52 06 PM
Screen Shot 2022-04-19 at 2 52 19 PM

### CODE REVIEW

File: utils/marketHours.ts [hunk 1]
```diff
@@ -10,16 +10,16 @@ import { utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
...
-			2: ['00:00:00', '89:00:00'],
-			3: ['00:00:00', '65:00:00'],
-			4: ['00:00:00', '41:00:00'],
-			5: ['00:00:00', '17:00:00'],
+			0: ['17:00:00', '126:00:00'], // Closing time is (24 * days until close inclusive) - 7
+			1: ['00:00:00', '112:00:00'], // ie. On Monday it will be 112 hours until market close (4pm Friday)
+			2: ['00:00:00', '88:00:00'],
+			3: ['00:00:00', '64:00:00'],
+			4: ['00:00:00', '40:00:00'],
+			5: ['00:00:00', '16:00:00'],
```
Comment (brossetti1) [inline utils/marketHours.ts:22]: matching - https://dune.com/joeyb/Synth-closures.  Basically Friday 8 PM UTC and open Sunday 9 PM UTC.

File: components/Badge/MarketBadge.tsx [hunk 2]
```diff
@@ -1,71 +1,59 @@
...
-	description: 'long' | 'short';
 };
 
-export const TimerBadge: FC<TimerBadgeProps> = ({ isOpen, description, timer }) => {
+export const TransitionBadge: FC<TransitionBadgeProps> = ({ isOpen }) => {
 	const { t } = useTranslation();
 
 	return (
 		<StyledBadge
 			background={isOpen ? theme.colors.common.primaryRed : theme.colors.common.primaryGreen}
```
Comment (Tburm) [inline components/Badge/MarketBadge.tsx:25]: I think these colors are confusing.

When a market is OPEN but CLOSING SOON we have a red badge. Red indicates to me that something is closed, or that I can't interact.

When a market is CLOSED but OPENING SOON we have a green badge. Green indicates to me that something is open and I can interact, but the market is closed.

I'd prefer if the "Closing soon" badge was yellow, and the "opening soon" badge is red.

**Review by Tburm — COMMENTED** [review]
Changes look good, but I'm not sure how to test the actual timer and closure times. Left a few nitpick comments on styling.

**Comment by brossetti1** [comment]
Screen Shot 2022-04-21 at 2 45 07 PM
Screen Shot 2022-04-21 at 2 45 15 PM

**Review by Tburm — APPROVED** [review]

**Pull request merged**